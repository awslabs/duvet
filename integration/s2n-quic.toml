name = "s2n-quic"
repo = "https://github.com/aws/s2n-quic"
version = "005f9461612df4f882b91844fa978a5ee603e9f8"
cmd = ["bash scripts/compliance"]
html_report = "target/compliance/report.html"
