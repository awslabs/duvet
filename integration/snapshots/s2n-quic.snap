version https://git-lfs.github.com/spec/v1
oid sha256:23cc03c36f7969188d61788fd08a51e1584fe9b6795d61b88f1962480e9e778f
size 6286485
